### DISCUSSION TITLE
Custom default handler

### DISCUSSION THREAD

**Discussion by patrickng-seek** [serverless-nextjs/serverless-next.js#2007] [Q&A]
There is `inputs.handler` but I am wondering how to use it, and how to call the `default-handler`. I copied the deprecated example which is not working, is there is a working example for current version?

```
const compat = require("@sls-next/next-aws-lambda");

module.exports = (page) => {
  const handler = (event, context, callback) => {
    // let's add some logging
    console.log("URL: ", event.path);

    // render page
    compat(page)(event, context, callback);

    console.log("This is cool (☞ﾟ∀ﾟ)☞");
  };
  return handler;
};
```

**Comment by dphang** [comment]
Hey there, sorry for the late response as I don't seem to get notifications for these and missed it. I am actually not sure as I think this was a community-contributed feature? Hopefully if you could mention which part of `inputs.handler` is not working we can try to get it to work again and possibly tested properly in the CI pipelines.
